ddtrace/tracer.py

```diff
@@ -145,22 +145,31 @@ def __init__(
         self.writer = writer  # type: TraceWriter
 
         # TRACER_PARTIAL_FLUSH_ENABLED should be deprecated after version 1.0.0 is released
-        partial_flush_enabled_default = asbool(get_env("tracer", "partial_flush_enabled", default=False))
+        partial_flush_enabled_deprecated_config = asbool(get_env("tracer", "partial_flush_enabled", default=False))
+        if partial_flush_enabled_deprecated_config:
+            log.warning(
+                """Please use DD_TRACE_PARTIAL_FLUSH_ENABLED to enable partial span flushing.
+                 DD_TRACER_PARTIAL_FLUSH_ENABLED config will be deprecated with version 1.0.0"""
+            )
```

Comment:
we have a helper for logging deprecations as "warnings" (instead of a warning log)

https://github.com/DataDog/dd-trace-py/blob/da34b2cd08936a660f0a835509b4d023343c4d92/ddtrace/utils/deprecation.py#L37-L49

Reply:
done